analysis/plotting.py

```diff
@@ -248,21 +256,69 @@ def write_violin_plot(self, benchmark_snapshot_df, image_path):
         self._write_plot_to_image(self.violin_plot, benchmark_snapshot_df,
                                   image_path)
 
-    def distribution_plot(self, benchmark_snapshot_df, axes=None):
+    def box_plot(self, benchmark_snapshot_df, axes=None, bugs=False):
```

Comment:
Would it make sense to `merge violin_plot()` and `box_plot()` into a single `box_or_violin_plot(violin=False)` which would draw violin/box depending on the parameter to remove some redundancy?

Reply:
+1

Reply:
combined!